hibernate-core/src/main/java/org/hibernate/metamodel/mapping/ordering/ast/SequencePart.java

```diff
@@ -16,6 +16,7 @@
 public interface SequencePart {
 	SequencePart resolvePathPart(
 			String name,
+			boolean isFormula,
```

Comment:
The added `isFormula` is not needed.  It is always `false`.

Reply:
Deleted it. Thanks for the comment.